services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/liitteet/AttachmentServiceImpl.kt

```diff
@@ -0,0 +1,96 @@
+package fi.hel.haitaton.hanke.liitteet
+
+import fi.hel.haitaton.hanke.HankeRepository
+import fi.hel.haitaton.hanke.currentUserId
+import fi.hel.haitaton.hanke.getCurrentTimeUTCAsLocalTime
+import java.util.*
+import javax.persistence.EntityNotFoundException
+import org.springframework.orm.jpa.JpaObjectRetrievalFailureException
+import org.springframework.transaction.annotation.Transactional
+import org.springframework.web.multipart.MultipartFile
+
+private val supportedFiletypes =
+    setOf(
+        "application/pdf",
```

Comment:
Would it be better to use values from https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/http/MediaType.html